<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoursePlanner_Test</class>
 <widget class="QMainWindow" name="CoursePlanner_Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoursePlanner_Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Busy Times</string>
    </property>
    <widget class="QComboBox" name="combo_courses">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_day">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_timeslot">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_add_busy">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>180</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Busy Times</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buton_busy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Busy Times</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_create">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Schedule</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_morning">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Morning</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Monday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thursday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Friday</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="table_afternoon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>611</width>
      <height>171</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Afternoon</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Monday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thursday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Friday</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Titillium Web&quot;;</string>
    </property>
    <property name="text">
     <string>WELCOME AUTOMATIC COURSE PLANNER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>Paste courses.csv file path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Trebuchet MS&quot;;</string>
    </property>
    <property name="text">
     <string>Paste services.csv file path</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
